API-Automation/test-suites/regression/pet/get_request.spec.js

```diff
@@ -6,26 +6,26 @@ describe('Search order', () => {
     let petId;
 
     beforeEach(async () => {
-        const response = await addNewPet(newPetData)
+        const response = await addNewPet(newPetData);
         petId = response.data.id;
-        return petId;
     });
 
     it('TC_10_1 User is able to find the pet by status', async () => {
-        const response = await findPetByStatus();
+        const response = await findPetByStatus(newPetData.status);
         expect(response.status).toBe(200);
         expect(_.isEqual(response.data, newPetData));
     });
 
     it('TC_10_2 User is able to find pet by Id', async () => {
-        const response = await findPetById();
+        const response = await findPetById(petId);
         expect(response.status).toBe(200);
         expect(_.isEqual(response.data, newPetData));
     });
-    
+
     afterEach(async () => {
         if (petId) {
```

Comment:
In all afterEach() methods, simply call await deletePet, deleteUser, etc. No need for checking if petId exists, if it does not exist, test will fail in it() block also.